packages/ui-devkit/src/compiler/types.ts

```diff
@@ -280,3 +368,7 @@ export interface BrandingOptions {
     largeLogoPath?: string;
     faviconPath?: string;
 }
+
+export interface AdminUiExtensionWithId extends AdminUiExtension {
```

Comment:
Not very elegant but that was the only way I found to make `id` required but not `pathAlias`.

Reply:
It's fine - it communicates the type and it's internal anyway.